playbooks/ha_log_cfg_sync.yaml

```diff
@@ -0,0 +1,138 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+---
+- name: Config rsync daemon on log server slave node
+  hosts: logserver-slave, zuul-scheduler-slave
+  become: yes
+  tasks:
+    - file: path=/etc/rsyncd state=directory
+
+    - file: path=/var/run/rsyncd.pid state=absent
+
+    - name: Create rsyncd config file for logs and zuul config files sync
+      template:
+        src: "{{ labkeeper_config_git_dest }}/rsync/rsyncd.conf.j2"
+        dest: /etc/rsyncd/rsyncd.conf
+
+    - name: Start rsync daemon on zuul slave node
+      shell: |
+        echo 'root:openlab' > /etc/rsyncd/rsyncd.pass
+        chmod 600 /etc/rsyncd/rsyncd.pass
+        rsync --daemon --config=/etc/rsyncd/rsyncd.conf
+      args:
+        executable: /bin/bash
+
+- name: Config rsync client on zuul master node
+  hosts: logserver-master, zuul-scheduler-master
+  become: yes
+  tasks:
+    - file: path=/etc/rsync state=directory
+
+    - name: Create sync password file
+      copy:
+        dest: /etc/rsync/rsync_client.pwd
+        content: "openlab"
```

Comment:
I will make this configurable with a inventory variable, but not sure if this need to be encrypted, we strictly limit with the `hosts allow` config option.

Reply:
thought again, maybe it is not need to concern about that, it is similar to the mysql password, we also exposed but only allow specific hosts to access. see: 
https://github.com/theopenlab/labkeeper/blob/master/playbooks/config-mysql.yaml#L73

Reply:
This will break base secure rule IMO even it works well. I'd like to say that Mysql is wrong as well. Just put my opinion here If you think it's OK, let it go. :)

Reply:
Can we generate a new uuid random key to apply for rsync password, like: metadata services token between nova and neutron?

Reply:
I am also not very sure, let's listen if there is other's opinion :)

Reply:
> Can we generate a new uuid random key to apply for rsync password, like: metadata services token between nova and neutron?

@kiwik surely we can, but not sure if it is necessary :), all the access we have limited to specific hosts, any other hosts cannot access.

Reply:
> > Can we generate a new uuid random key to apply for rsync password, like: metadata services token between nova and neutron?
> 
> @kiwik surely we can, but not sure if it is necessary :), all the access we have limited to specific hosts, any other hosts cannot access.

Avoiding password in source code is always better choice, right? :)